scripts/run.sh

```diff
@@ -112,7 +112,7 @@ INPUT_PREFIX=${INPUT_DIR/\/*/}
 TAG=${INPUT_DIR/${INPUT_PREFIX}\//}
 INPUT_DIR=${BASEDIR}/EVGEN/${TAG}
 mkdir -p ${INPUT_DIR}
-TAG=${DETECTOR_VERSION:-main}/${DETECTOR_CONFIG}/${TAG}
+TAG=${DETECTOR_VERSION:-main}/${DETECTOR_CONFIG}/${MISC_CONFIG:+${MISC_CONFIG}/}${TAG}
```

Comment:
If you put it on the TAG level, maybe plan ahead and use
```suggestion
TAG=${DETECTOR_VERSION:-main}/${DETECTOR_CONFIG}/${TAG_PREFIX:+${TAG_PREFIX}/}${TAG}${TAG_SUFFIX:+/${TAG_SUFFIX}}
```